### PR TITLE
Setting up prettier pre-commit hook

### PR DESCRIPTION
**Pull request by Rucha1499**
- Setting up prettier pre-commit hook
- Modifications in README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,12 @@ https://welcome.realdevsquad.com/discord
 # Code of conduct at RDS
 
 https://welcome.realdevsquad.com/code-of-conduct.html
+
+# Running the project
+
+- Run `npm install` after opening the repository in your editor. This will install all the necessary packages for the project which are mentioned in the `package.json` file of the project.
+
```
Comment (swarajpure) [inline README.md:26]: Can we please link `CONTRIBUTING.md` here? The first 3 steps are related to the step up itself
Once the devs have the setup done, we can ask them to do `npm install`

File: README.md [hunk 2]
```diff
@@ -19,3 +19,12 @@ https://welcome.realdevsquad.com/discord
...
+# Scripts used in project
+
+- `npm run check` checks the formatting of all the files in the project. This script runs in a pre-commit hook. It will warn about the improper formatting present in the file.
+- `npm run fix` can be run for fixing the formatting before commiting your changes.
```
Comment (swarajpure) [inline README.md:30]: We can also mention that if the devs get an error during commiting, they can use this script to get rid of the erros.

File: package.json [hunk 3]
```diff
@@ -11,6 +11,10 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "pre-commit": "^1.2.2",
     "prettier": "2.2.1"
-  }
+  },
+  "pre-commit": [
+    "check"
+  ]
```
Comment (swarajpure) [inline package.json:19]: Good to have it!

**Review by swarajpure — COMMENTED** [review]
Great one!

Please check the comments.

File: README.md (hunk 1 above)
Comment (Rucha1499) [reply README.md:26]: Sure. It will be of great help! 👍

File: README.md (hunk 2 above)
Comment (Rucha1499) [reply README.md:30]: Okay!!

**Review by swarajpure — APPROVED** [review]
Great job!

**Pull request merged**